docs/api.rst

```diff
@@ -444,12 +444,12 @@ Querying general network events
 
 
 You can query for non-channel specific events by making a ``GET`` request to the
-endpoint of the token registry contract. ``GET /api/<version>/events/network/<token_registry_address>``
+following endpoint. ``GET /api/<version>/events/network``
```

Comment:
Will this endpoint return the events of all know registries or only one?

- If we are returning the events from all registries the URL must use plural
- If we are not, then I think we should document this behavior and inform the user we have a default registry.

Note: Forcing a default registry is not future-proof.

Also, the sentence `You can query for non-channel specific events` is misleading, since this endpoint seems to only return `Registry` events but not `ChannelManager` events, can we improve on that?